lib/ansible/config/base.yml

```diff
@@ -930,6 +930,9 @@ DEFAULT_SQUASH_ACTIONS:
   - {key: squash_actions, section: defaults}
   type: list
   version_added: "2.0"
+  deprecated:
+    why: In favor of supplying a list directly to the name parameter
+    version: "2.9"
```

Comment:
`alternative: ` though, does `ansible-doc` support this?

Reply:
I specifically didn't include `alternative` as I couldn't get the message to sound reasonable.  `alternative` in this case would indicate that there is an alternative config, but there isn't.

The `why` seems to describe why and the alternative, without generating a message that isn't grammatically understandable.

Reply:
alternative is 'pass list directly to module'

Reply:
I'll have to play around with it.  I tried adding `alternative` but the message that was constructed never made sense grammatically, so I removed it.